docs/reference/benchmarks.md

```diff
@@ -12,31 +12,33 @@ This page describes each benchmark that is part of fuzzbench.
-| harfbuzz-1.3.2              | fuzz-target            | False           | 58                    | 10021                    | 6.24                 |
-| jsoncpp_jsoncpp_fuzzer      | jsoncpp_fuzzer         | True            | 0                     | 5536                     | 5.96                 |
-| lcms-2017-03-21             | fuzz-target            | False           | 1                     | 6959                     | 6.11                 |
-| libjpeg-turbo-07-2017       | fuzz-target            | False           | 1                     | 9586                     | 6.4                  |
-| libpcap_fuzz_both           | fuzz_both              | False           | 0                     | 8149                     | 6.14                 |
-| libpng-1.2.56               | fuzz-target            | False           | 1                     | 2991                     | 5.8                  |
-| libxml2-v2.9.2              | fuzz-target            | False           | 0                     | 50461                    | 8.23                 |
-| mbedtls_fuzz_dtlsclient     | fuzz_dtlsclient        | False           | 1                     | 10942                    | 6.67                 |
-| openssl_x509                | x509                   | True            | 2241                  | 45989                    | 18.14                |
-| openthread-2019-12-23       | fuzz-target            | False           | 0                     | 17932                    | 6.72                 |
-| php_php-fuzz-parser         | php-fuzz-parser        | True            | 2782                  | 123767                   | 15.57                |
-| proj4-2017-08-14            | fuzz-target            | False           | 44                    | 6156                     | 6.17                 |
-| re2-2014-12-09              | fuzz-target            | False           | 0                     | 6547                     | 6.02                 |
-| sqlite3_ossfuzz             | ossfuzz                | True            | 1258                  | 45136                    | 7.9                  |
-| systemd_fuzz-link-parser    | fuzz-link-parser       | False           | 6                     | 53453                    | 5.91                 |
-| vorbis-2017-12-11           | fuzz-target            | False           | 1                     | 5022                     | 6.0                  |
-| woff2-2016-05-06            | fuzz-target            | False           | 62                    | 10923                    | 6.72                 |
-| zlib_zlib_uncompress_fuzzer | zlib_uncompress_fuzzer | False           | 0                     | 875                      | 5.69                 |
+| Benchmark                   | Fuzz target            | Has dictionary? | Is stable? | Seed type\*              | Number of seed inputs  | Number of progam edges\*\*| Binary Size (MB)\*\*\*|
```

Comment:
ehhh...I'd rather have a percentage if possible but this is a good start.

Reply:
+1, can either stable column or add percents, i dont think all other targets are fully unstable.

Reply:
well anything less than 100% is unstable, and 14 benchmarks are.
I ran the experiments for ~15 minutes and usually the stability increases over time (instability is often hash maps or threads and they happen with most fuzz attempts) as the coverage increases over time, increasing the number of stable map entries vs the unstable ones.
so if you want to have numbers you need them from a full experiment. you can find them in any fuzzer_stats file of any afl variant of an already run experiment.
And I must say I deleted my data. I dont build in the cloud but on my machine and I do not have endless disk space... plus the building takes ages and hogs my cpu ... :)

Reply:
Stability column, we will validate and add after 2/3 weeks. Can you please remove it from this PR.